test/test_scan.py

```diff
@@ -1,11 +1,14 @@
 import sys
 import unittest
-import torch_xla
+from functools import reduce
+
 import torch
+from torch.utils._pytree import tree_map, tree_flatten, tree_iter, tree_leaves, PyTree
+
+import torch_xla
 from torch_xla.experimental.scan import scan
-from torch.utils._pytree import tree_map, tree_flatten, tree_iter
 
-from test_utils import XlaTestCase
+from test_utils import XlaTestCase  # type:ignore
```

Comment:
did some formatter complained?

Reply:
PyLance can't find this class for some reason.